tests/test_by_date.py

```diff
@@ -73,7 +73,7 @@ def test_by_date_json(by_date: ByDate) -> None:
     start_date = "2022-01-03"
```

Comment:
if `tests/data_mini/data-2022.json` is mini then why is it too big for diff still?

Reply:
we have only 179 signals in it